src/Analyzer.JsonRuleEngine.UnitTests/ExpressionConverterTests.cs

```diff
@@ -42,6 +42,10 @@ public class ExpressionConverterTests
         [DataRow("equals", "someString", "exists", true, DisplayName = "{\"Equals\": \"someString\"}, Where: {\"Exists\": true}")]
         [DataRow("notEquals", 0, DisplayName = "{\"NotEquals\": 0}")]
         [DataRow("regex", "regexPattern", DisplayName = "{\"Regex\": \"regexPattern\"}")]
+        [DataRow("greater", 100, DisplayName = "{\"Greater\": 100}")]
+        [DataRow("less", 100, DisplayName = "{\"Less\": 100}")]
+        [DataRow("greaterOrEquals", 100, DisplayName = "{\"GreaterOrEquals\": 100}")]
+        [DataRow("lessOrEquals", 100, DisplayName = "{\"LessOrEquals\": 100}")]
         [DynamicData(nameof(InOperatorTestScenarios), DynamicDataDisplayName = nameof(GetInOperatorTestDisplayName))]
         public void ReadJson_LeafWithValidOperator_ReturnsCorrectTypeAndValues(string operatorProperty, object operatorValue, params object[] whereCondition)
```

Comment:
Add tests below in the invalid cases too.

Reply:
I'm not sure if this is an outdated comment or I'm not seeing the missing test cases

Reply:
I tried but at that stage it doesn't fail with invalid values like that

Reply:
Oh, that makes sense, I guess those tests would be in the operator tests, not in Expression conversion.